shell/platform/android/io/flutter/embedding/engine/FlutterJNI.java

```diff
@@ -212,7 +217,12 @@ public boolean isAttached() {
   public void attachToNative(boolean isBackgroundView) {
     ensureRunningOnMainThread();
     ensureNotAttachedToNative();
-    nativePlatformViewId = nativeAttach(this, isBackgroundView);
+    nativePlatformViewId = performNativeAttach(this, isBackgroundView);
+  }
+
+  @VisibleForTesting
```

Comment:
Can this be package private?

Reply:
Unfortunately, it can't because `FlutterJNI` is a dependency of the `PlatformViewsController`, which for historic reasons is a different package.